### PR TITLE
fix: remove mouse wheel support from show sdk & show scroll options in help view

### PR DESCRIPTION
**Pull request by k3llymariee**
Removes mouse wheel support from the viewport, because with this enabled we're unable to highlight the text within to copy/paste (which is kinda the whole point of it). 

image

### CODE REVIEW

File: internal/quickstart/show_sdk_instructions.go [hunk 1]
```diff
@@ -51,16 +51,21 @@ func NewShowSDKInstructionsModel(
 		BorderForeground(lipgloss.Color("62")).
 		PaddingRight(2)
 
+	h := help.New()
+	h.ShowAll = true
```
Comment (sunnyguduru) [inline internal/quickstart/show_sdk_instructions.go:55]: Did we define help ourselves somewhere?

**Review by sunnyguduru — APPROVED** [review]

**Review by dbolson — APPROVED** [review]
The full version of the help with fewer elements looks kind of weird. It could be good enough without redoing the entire help, but do you think defaulting to short help and showing "more" would hide the fact that you can scroll too much?

**Comment by k3llymariee** [comment]
@dbolson yeah I worry about hiding those keys/making it less obvious that it's scrollable. I don't hate it as-is 😅 but maybe we can wait for more feedback

File: internal/quickstart/show_sdk_instructions.go (hunk 1 above)
Comment (k3llymariee) [reply internal/quickstart/show_sdk_instructions.go:55]: this is just the bubbletea Help model

**Pull request merged**